kieker-monitoring/build.gradle

```diff
@@ -40,6 +40,14 @@ dependencies {
 	compile resolveLibrary('framework-libs/spring/spring-core-*.jar')
 	compile resolveLibrary('framework-libs/spring/spring-web-*.jar')
 	compile resolveLibrary('framework-libs/spring/spring-webmvc-*.jar')
+	
```

Comment:
Shouldn't those be `testCompile`s?

Reply:
I am not sure. They were like this before. Also resolveLibrary is used. We are on the way to get rid of those local libraries.